requirements_dev.in

```diff
@@ -5,7 +5,7 @@
 # to update `requirements_dev.txt` and your current Python environment.
 Pillow==10.1.0
 certifi==2023.7.22
-coverage==7.2.3
+coverage==7.8.0
```

Comment:
Just to check, we can't use 7.2.3 (which seems to have fewer problems) with py3.10?

Reply:
Ah ok. And, just to confirm, 7.2.3 still leads to the same coverage issues that necessitate the exclusion of jobs/ files?